.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+use Nexus\CsConfig\Factory;
+use PhpCsFixer\Finder;
+use Utils\PhpCsFixer\CodeIgniter4;
+
+$finder = Finder::create()
+    ->files()
+    ->in([
+        __DIR__ . '/app', // @todo relocate when `header_comment` is enabled
+        __DIR__ . '/public', // @todo relocate when `header_comment` is enabled
+        __DIR__ . '/system',
+        __DIR__ . '/tests',
+        __DIR__ . '/utils',
```

Comment:
The https://github.com/codeigniter4/CodeIgniter4/blob/develop/admin/starter/app/Config/Paths.php need to be added as well?

Reply:
Done.